subxt/src/blocks/extrinsic_types.rs

```diff
@@ -558,6 +671,101 @@ impl<T: Config> ExtrinsicEvents<T> {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct ExtrinsicSignedExtensions<'a, T: Config> {
```

Comment:
Is there any use for having `T: Config` here? I don't think so at the moment, so maybe it can be removed from this and the child structs at the mo?